io.github.ec_.Quake3e.OpenArena.yaml

```diff
@@ -11,12 +11,6 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --socket=fallback-x11
-  # Needs a persistent location for Quake 3 data files, the user
-  # has permission to add things to and this is the only one in Quake3e's
-  # search path.
-  - --persist=.openarena
```

Comment:
This change just caused the current users to lose their data folder. You should add a migration support for existing users.